src/applications/disability-benefits/all-claims/content/addDisabilities.jsx

```diff
@@ -27,3 +29,42 @@ export const uiDescription = (
     </AdditionalInfo>
   </div>
 );
+
+const alertContent = newOnly => (
+  <>
+    <p>
+      You’ll need to add a new disability or choose a rated disability to claim.
+      We can’t process your claim without a disability selected. Please add a
+      new disability or choose a rated disability for increased compensation.
+    </p>
+    {!newOnly && (
```

Comment:
@katelyncai should we have a link for the 'new only' alert that takes users back to the 'new or increase' selection page, or just leave it like this and omit the link in the 'new only' flow's alert box?

Reply:
🤔 this also doesn't show when you select both new & rated options.